apps/main/src/loan/components/PageLoanCreate/LoanFormCreate/index.tsx

```diff
@@ -393,12 +393,12 @@ const LoanCreate = ({
               formValues.debtError === 'too-much' ? t`Amount is greater than ${formatNumber(maxRecv)}` : undefined
             }
             disabled={disabled}
-            maxBalance={{
+            walletBalance={{
```

Comment:
I started changing this to `walletBalance`, but then found that this `LargeTokenInput` won't be shown at all, because if you're in beta mode you're seeing a new loan-create-tab altogether, the same holds for the lend app. So essentially this is dead code?